pandas/tools/merge.py

```diff
@@ -1258,9 +1258,12 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
     join_axes : list of Index objects
         Specific indexes to use for the other n - 1 axes instead of performing
         inner/outer set logic
-    verify_integrity : boolean, default False
```

Comment:
there is a doc-string in ``merge.rst``, can you make sure to update that as well.